4.classify-single-cell-phenotypes/classification_utils.py

```diff
@@ -0,0 +1,135 @@
+"""
+utils for classifying cells from Cell Health Data
+"""
+
+import pathlib
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LogisticRegression
+
+
+def get_probas_dataframe(
+    plate_features: pd.DataFrame, model: LogisticRegression, feature_type: str
+) -> pd.DataFrame:
+    """
+    Get probabilites for plate features from a phenotypic classification model
+
+    Parameters
+    ----------
+    plate_features : pd.DataFrame
+        plate features to classify
+    model : LogisticRegression
+        model to use for plate feature classification
+    feature_type : str
+        type of features to use for classification.
+        CP, DP, or CP_and_DP
+
+    Returns
+    -------
+    pd.DataFrame
+        dataframe with single-cell probabilities for classes from given model
+    """
+
+    # determine which feature columns should be loaded depending on feature type
+    # if there is no "and" we can use feature type as prefix
+    if "and" not in feature_type:
```

Comment:
Consider using a specific check for the literal string value which includes "and" (there appears to be only one circumstance where this occurs as-is).
```suggestion
    if feature_type != "CP_and_DP":
```